packages/create-block-tutorial-template/templates/src/index.js.mustache

```diff
@@ -37,10 +37,11 @@ registerBlockType( '{{namespace}}/{{slug}}', {
 		},
 	},
 
+    // The edit and save properties below are using object property shorthand. See https://alligator.io/js/object-property-shorthand-es6/
```

Comment:
Do we still need to document language features added in 2015?

In other places we use more complex syntax without any comments, example:
```jsx
<p { ...useBlockProps.save() }>
```

Reply:
Based on #33613 it seems that there are developers for whom this is new/confusing. Coming from a JavaScript background, this is probably not required but there are a lot of PHP-first developers for whom much ( perhaps all ) of it is new. If this was production code, I wouldn't be pushing for these kinds of changes but these templates are used by people who are trying to learn how to create blocks and as such, I think we have some responsibility to provide a little help explaining concepts that, while introduced in 2015, are potentially new to developers not immersed in JavaScript.

FWIW I'd also advocate for adding a link to the object spread syntax as well.